src/query/service/tests/it/storages/fuse/operations/mutation/segments_compact_mutator.rs

```diff
@@ -82,6 +82,7 @@ async fn test_compact_segment_normal_case() -> Result<()> {
     append_rows(ctx.clone(), num_inserts).await?;
 
     // check count
+    ctx.evict_table_from_cache("default", "default", "t")?;
```

Comment:
Why need to call `evict_table_from_cache` here?

Reply:
see line 82 ,it has a append_rows, we need to update cache. You can see the summary session 2 of this pr. cc @BohuTANG